src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -412,6 +412,17 @@ internal void LoadResponse(FunctionLoadResponse loadResponse)
             {
                 //Cache function load errors to replay error messages on invoking failed functions
                 _functionLoadErrors[loadResponse.FunctionId] = ex;
+
+                FunctionMetadata metadata = _functions.SingleOrDefault(p => p.FunctionId == loadResponse.FunctionId);
+                if (metadata != null)
+                {
+                    _loggerFactory.CreateLogger(LogCategories.CreateFunctionCategory(metadata.Name)).LogError(ex, "Function load error.");
```

Comment:
@pragnagopa FYI -- here's the line that actually fixes the issue at startup.

Reply:
Same thing as elsewhere -- because we need to explicitly create the category, we need the logger factory.